Samples/AzureWebSample/OrleansAzureSilos/OrleansConfiguration.xml

```diff
@@ -1,10 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
+<!-- 
+	This is a sample silo configuration file for use by an Azure worker role acting as an Orleans silo. 
+	The comments illustrate common customizations.
+	Elements and attributes with no comments should not usually need to be modified.
+	For a detailed reference, see "Orleans Configuration Reference.html".
+-->
 <OrleansConfiguration xmlns="urn:orleans">
   <Globals>
     <Liveness LivenessType="AzureTable" />
   </Globals>
 	<Defaults>
-    <Tracing DefaultTraceLevel="Info" TraceToConsole="true" TraceToFile="{0}-{1}.log" >
+	  <!-- 
+	  To turn tracing off, set DefaultTraceLevel="Off" and have no overrides. To see a minimum of messages, set DefaultTraceLevel="Error".
+      For the trace log file name, {0} is the silo name and {1} is the current time. 
+	  Setting WriteTraces to true will cause detailed performance information to be collected and logged about the individual steps in the
+	  message lifecycle. This may be useful debugging performance issues.
+	  -->
+    <Tracing DefaultTraceLevel="Info" TraceToConsole="true" TraceToFile="none" >
```

Comment:
Wouldn't it be better to leave TraceToFile, so that it's easier to debug in the emulator or when WAD isn't initialized? I consider removing TraceToFile a potential minor performance optimization. I'm afraid it may end up being counterproductive in the sample.